cmd/ocm/main.go

```diff
@@ -38,13 +39,14 @@ import (
 	"github.com/openshift-online/ocm-cli/cmd/ocm/login"
 	"github.com/openshift-online/ocm-cli/cmd/ocm/logout"
 	"github.com/openshift-online/ocm-cli/cmd/ocm/patch"
+	plugincmd "github.com/openshift-online/ocm-cli/cmd/ocm/plugin"
 	"github.com/openshift-online/ocm-cli/cmd/ocm/post"
 	"github.com/openshift-online/ocm-cli/cmd/ocm/token"
 	"github.com/openshift-online/ocm-cli/cmd/ocm/tunnel"
 	"github.com/openshift-online/ocm-cli/cmd/ocm/version"
 	"github.com/openshift-online/ocm-cli/cmd/ocm/whoami"
 	"github.com/openshift-online/ocm-cli/pkg/arguments"
-	"github.com/openshift-online/ocm-cli/pkg/plugin"
+	plugin "github.com/openshift-online/ocm-cli/pkg/plugin"
```

Comment:
cosmetic: consider renaming the directory to pkg/plugins so we won't need import renaming.  your call.

Reply:
actually ignore that.

@igoihman @vkareh Do you have an opinion on the command naming itself?  `ocm plugin list` vs `ocm plugins list` vs `ocm list plugins` (may be confusing because "ocm list foo" is for API objects not local stuff) vs just `ocm plugins`?
I recall somebody had an opinion on verb/noun order but can't find it...

Reply:
yes, regarding API resources, we're trying to standardize on `ocm {verb} {noun}`, which would make this `ocm list plugins`, but you bring up a good point: this is not an API resource, so maybe it's ok to break convention??

Also, cobra supports aliases, so I'd make the default to be plural `plugins` (since it's a list of many) with an alias to `plugin`.